css/css/style.css

```diff
@@ -0,0 +1,15 @@
+@import url(variables.css);
+@import url(navbar.css);
+@import url(content.css);
+@import url(footer.css);
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  /* overflow: hidden; */
```

Comment:
remove commented code

Reply:
Made the said changes.. thanks for reviewing